services/ir_pagerduty.rb

```diff
@@ -29,11 +29,11 @@ def attach_contact_methods_to_user(user)
   end
 
   def fetch_todays_schedules_users(sid)
-    today = Date.today
```

Comment:
@s-block Hey dude, have a couple of questions/suggestions regarding this PR.

- I'd recommend using a constant with a useful name to define the starting time for the work day and end of work day.
- Why are we using `URI.escape` here? Date.today shouldn't return anything with characters that the default REGEXP::UNSAFE would match on.
- This was raised as a bug but we don't have any regression test?

We could refactor this to something like..(just off the top of my head so might not be 100% valid)

```ruby
START_OF_WORKING_DAY = 'T10:01Z'.freeze
END_OF_WORKING_DAY = 'T16:59Z'.freeze

def fetch_todays_schedules_users(schedule_id)
 # Use of fetch is questionable here, why would users not exist?
  # Added as there was a rescue in original.
  self.Schedule.users(schedule_id, since_date, until_date).
    fetch('users', []).
    map(&method(:attach_contact_methods_to_user))
end

private

def since_date
  "#{Date.today + START_OF_WORKING_DAY}"
end

def until_date
  "#{Date.today + END_OF_WORKING_DAY}"
end
```

cc @andyhd